src/main/java/com/sap/sgs/phosphor/fosstars/data/github/PackageManagement.java

```diff
@@ -0,0 +1,232 @@
+
+import static com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures.LANGUAGES;
+import static com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures.PACKAGE_MANAGERS;
+import static com.sap.sgs.phosphor.fosstars.model.value.Language.C_SHARP;
+import static com.sap.sgs.phosphor.fosstars.model.value.Language.F_SHARP;
+import static com.sap.sgs.phosphor.fosstars.model.value.Language.JAVA;
+import static com.sap.sgs.phosphor.fosstars.model.value.Language.JAVASCRIPT;
+import static com.sap.sgs.phosphor.fosstars.model.value.Language.PHP;
+import static com.sap.sgs.phosphor.fosstars.model.value.Language.PYTHON;
+import static com.sap.sgs.phosphor.fosstars.model.value.Language.RUBY;
+import static com.sap.sgs.phosphor.fosstars.model.value.Language.SCALA;
+import static com.sap.sgs.phosphor.fosstars.model.value.Language.VISUALBASIC;
+import static com.sap.sgs.phosphor.fosstars.model.value.PackageManager.COMPOSER;
+import static com.sap.sgs.phosphor.fosstars.model.value.PackageManager.DOTNET;
+import static com.sap.sgs.phosphor.fosstars.model.value.PackageManager.GRADLE;
+import static com.sap.sgs.phosphor.fosstars.model.value.PackageManager.MAVEN;
+import static com.sap.sgs.phosphor.fosstars.model.value.PackageManager.NPM;
+import static com.sap.sgs.phosphor.fosstars.model.value.PackageManager.PIP;
+import static com.sap.sgs.phosphor.fosstars.model.value.PackageManager.RUBYGEMS;
+import static com.sap.sgs.phosphor.fosstars.model.value.PackageManager.YARN;
+
+import com.sap.sgs.phosphor.fosstars.model.Value;
+import com.sap.sgs.phosphor.fosstars.model.ValueSet;
+import com.sap.sgs.phosphor.fosstars.model.value.Language;
+import com.sap.sgs.phosphor.fosstars.model.value.Languages;
+import com.sap.sgs.phosphor.fosstars.model.value.PackageManager;
+import com.sap.sgs.phosphor.fosstars.model.value.PackageManagers;
+import com.sap.sgs.phosphor.fosstars.model.value.ValueHashSet;
+import java.io.IOException;
+import java.util.EnumMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.Predicate;
+import org.kohsuke.github.GHContent;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GitHub;
+
+/**
+ * This data provider returns package managers which are used in a project.
+ */
+public class PackageManagement extends AbstractGitHubDataProvider {
+
+  /**
+   * A minimal number of characters in a config to consider it valid.
+   */
+  private static final int ACCEPTABLE_CONFIG_SIZE = 100;
+
+  /**
+   * Maps a programming language to its package manager.
+   */
+  private static final Map<Language, PackageManagers> KNOWN_PACKAGE_MANAGERS
+      = new EnumMap<>(Language.class);
+
+  static {
+    register(JAVA, MAVEN, GRADLE);
+    register(SCALA, MAVEN);
+    register(JAVASCRIPT, NPM, YARN);
+    register(C_SHARP, DOTNET);
+    register(F_SHARP, DOTNET);
+    register(VISUALBASIC, DOTNET);
+    register(PYTHON, PIP);
+    register(RUBY, RUBYGEMS);
+    register(PHP, COMPOSER);
```

Comment:
If you were looking for some package managers for Go https://github.com/golang/go/wiki/PackageManagementTools

Reply:
Good to know. In fact, the data provider don't cover all existing package managers. Currently it focuses only on those which are supported by the security alerts on GitHub. That's going to be used in the dependency scans score in the future. Unfortunately, the security alerts don't support Go.